server/lib/subscriptions.js

```diff
@@ -190,16 +191,21 @@ export async function handleRetryStatus(order, transaction) {
 export function getNextChargeAndPeriodStartDates(status, order) {
   const initial = order.Subscription.nextPeriodStart || order.Subscription.createdAt;
   let nextChargeDate = moment(initial);
+  const dayOfTheMonth = nextChargeDate.date();
   const response = {};
   if (status === 'new' || status === 'success') {
-    if (order.Subscription.interval === 'month') {
+    if (order.Subscription.interval === intervals.MONTH) {
       nextChargeDate.add(1, 'months');
-    } else if (order.Subscription.interval === 'year') {
+    } else if (order.Subscription.interval === intervals.YEAR) {
       nextChargeDate.add(1, 'years');
     }
-    if (status === 'new') {
+
+    if (status === 'new' && order.Subscription.interval === intervals.MONTH && dayOfTheMonth >= 15) {
```

Comment:
```suggestion
    if (status === 'new' && order.Subscription.interval === intervals.MONTH && dayOfTheMonth > 15) {
```

Would do after the 15.